src/WebJobs.Script.WebHost/Extensions/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Extensions;
+
+namespace WebJobs.Script.WebHost.Extensions
```

Comment:
We have an `HttpRequestExtensions` class in Script, would be good to just move this there.

Reply:
done